src/ngraph/runtime/cache.cpp

```diff
@@ -15,21 +15,22 @@
 //*****************************************************************************
 
 #include "ngraph/runtime/cache.hpp"
+#include "ngraph/env_util.hpp"
 
 using namespace ngraph;
 using namespace std;
 
 // Constructor
 runtime::LRUCache::LRUCache()
 {
-    char* cache_size = getenv("NGRAPH_CACHE_SIZE");
-    if (cache_size == nullptr)
+    int32_t cache_size = getenv_int("NGRAPH_CACHE_SIZE");
+    if (cache_size <= 0)
```

Comment:
shouldn't this be < 0  and not <=0? ....lets assume the user wants to set cache size of 0 and not use the cache...in this case it will set it to 1024

Reply:
@nbpatel I feel runtime::LRUCache::add_entry needs to be modified if we allow cache_size to be 0, WDYT?